conjure-java-core/src/test/resources/example-service.yml

```diff
@@ -68,6 +68,8 @@ services:
             param-id: Test-Header
             param-type: header
             type: string
+            markers:
+              - Safe
```

Comment:
We might want a test with two markers on the same parameter

Reply:
done